packages/@react-spectrum/color/src/ColorSlider.tsx

```diff
@@ -37,22 +37,21 @@ function ColorSlider(props: SpectrumColorSliderProps, ref: FocusableRef<HTMLDivE
   let trackRef = useRef();
   let domRef = useFocusableRef(ref, inputRef);
 
-  // The default label should be localized...
-  let defaultLabel = channel[0].toUpperCase() + channel.slice(1);
-
   let labelText = props.label;
-  let ariaLabel = props['aria-label'] ?? (labelText == null ? defaultLabel : undefined);
 
   let numberFormatter = useNumberFormatter();
   let state = useColorSliderState({...props, numberFormatter});
   let {inputProps, thumbProps, groupProps, trackProps, labelProps, gradientProps} = useColorSlider({
     ...props,
-    label: labelText,
-    'aria-label': ariaLabel,
+    'aria-label': channel,
```

Comment:
As an aside, for vertical color sliders that have a `label` provided, should the `aria-label` include the label text as well? The label element is omitted for vertical color sliders so the `aria-labelledby` that useLabel returns partially points to a non-existent element.  

This can be handled separately, doesn't seem in scope of the pull but figured I should ask

Reply:
That may be a good idea to ensure that the labelling intent is preserved for the vertical use case.

I also now realize that I'm overriding any `aria-label` from props with the `channel` value, which should not be done. We should only pass the `channel` when no `aria-label` is defined.